### ISSUE TITLE
Make releases less noisy

### ISSUE THREAD

**Issue by troessner** [troessner/reek#1392]
Since we are explicitly putting version numbers in our documentation links our releases have become very noisy as you can see in #1391.
Ideally, there would be one cucumber feature that ensures that our links change as well with new versions and that would then cause a one line change per new version release.

**Comment by mvz** [comment]
Agreed. Perhaps we can add some interpolation in the expected strings in the feature files.

**Comment by eloyesp** [comment]
As documentation is bundled with the gem, links could point to the local filesystem, that way you don't need github nor version number and also don't need internet.

The documentation can also be provided inline (with a flag or may be a custom command).

~~~
app/controllers/users_controller.rb -- 33 warnings:
  [48, 50, 54, 56]:DuplicateMethodCall: UsersController#add_admin_role calls '@user.name' 4 times [ reek doc duplicate_method_call ]

$ reek doc duplicate_method_call
# Duplicate Method Call

Markdown here
~~~

**Comment by troessner** [comment]
That is a very good idea, thank you for that :)
We might come back to this in the future.
